docs/api/rest/config.md

```diff
@@ -33,12 +33,14 @@ curl --location --request GET 'http://localhost:5000/config/ae'
   {
     "name": "brain-tumor",
     "aeTitle": "BrainTumorModel",
-    "applications": ["brain-tumor", "b75cd27a-068a-4f9c-b3da-e5d4ea08c55a"]
+    "workflows": ["brain-tumor", "b75cd27a-068a-4f9c-b3da-e5d4ea08c55a"],
+    "grouping": "0010,0020",
+    "ignoredSopClasses": ["1.2.840.10008.5.1.4.1.1.1.1"]
```

Comment:
Maybe we should consider specifying the expected SOP Class(es) instead of ignored/disallowed, simply because there are many storage classes to be added in the list to make it close to complete. A workflow typically accepts only a single or a few SOP classes.